src/www/widgets/widgets/thermal_sensors.widget.php

```diff
@@ -267,8 +267,7 @@ function getBoolValueFromConfig(&$configArray, $valueKey, $defaultValue)
 		</tr>
 		<tr>
 		<td colspan="3">
-      <span>* <?= gettext('You can configure a proper Thermal Sensor / Module under <br />
-    &nbsp;&nbsp;&nbsp;<a href="system_advanced_misc.php">System &gt; Advanced &gt; Miscellaneous : Thermal Sensors section</a>.') ?></span>
+      <span>* <?= sprintf(html_safe(gettext('You can configure a proper Thermal Sensor / Module under %s%sSystem > Advanced > Miscellaneous : Thermal Sensors section%s.')),'<br />&nbsp;&nbsp;&nbsp;','<a href="system_advanced_misc.php">','</a>') ?></span>
```

Comment:
No more breaking this into two lines + drop implication of "System: Advanced: Misc" as it hard to translate consistently and may be subject to change.